php/Terminus/Models/TerminusModel.php

```diff
@@ -6,25 +6,25 @@
 
 abstract class TerminusModel
 {
-  /**
-   * @var string
-   */
+    /**
+     * @var string
+     */
     public $id;
-  /**
-   * @var array Arguments for fetching this model's information
-   */
+    /**
+     * @var Request
+     */
+    public $request;
```

Comment:
This has been worked around by applying RequestAwareTrait&Interface to TerminusModel and TerminusCollection.